### DISCUSSION TITLE
Plaid Integration

### DISCUSSION THREAD

**Discussion by greenejob** [andrewdwallo/erpsaas#84] [General]
Hi everyone,

I’m exploring the Plaid integration and wondering if there’s a way for individuals or non-companies to obtain credentials to download transactions. The developer setup seems very company-focused, and I’m not sure if there’s an alternative for those who don’t operate as a business entity.

Does anyone know if Plaid supports individual users in this context, or is there a workaround to achieve this?

In general, going through their signup process and doing a full corporate diligence just to access your own accounts seems to be a bit much.

Thanks in advance for your help!

**Comment by andrewdwallo** [comment]
@greenejob Yes you can complete the production registration form as an individual/sole proprietor, you don't need to operate as a business entity. Not sure if you are in Plaid's discord channel but their moderator answered a similar question of mine that relates to this: [Discord message link](https://discord.com/channels/986328701488758824/1022606186580815892/1210467871646289941)

**Reply by greenejob** [reply]
@andrewdwallo I see you had the same question many months ago. On to the security diligence questionnaire which again assumes that this is being used for large-scale data.
Thanks again for your help!
